packages/calypso-e2e/src/jest-playwright-config/environment.ts

```diff
@@ -276,7 +182,9 @@ class JestEnvironmentPlaywright extends NodeEnvironment {
 			console.error( `Artifacts for ${ this.testFilename }: ${ this.testArtifactsPath }` );
 		}
 		// Regardless of pass/fail status, close the browser instance.
-		return await this.global.browser.close();
+		await this.global.browser.close();
+
+		await super.teardown();
```

Comment:
`super.teardown` was always present, but it has been moved to **after** our custom teardown steps.